README.md

```diff
@@ -4,20 +4,29 @@ You can deploy 1Password SCIM Bridge on any supported infrastructure that allows
 
 ## Automatic (one-click) deployment
 
-The easiest way to deploy 1Password SCIM Bridge is with our one-click apps, which are currently available for Google Cloud Platform and DigitalOcean:
+For a guided deployment, you can use our Google Cloud or DigitalOcean marketplace app to deploy 1Password SCIM Bridge:
 
 - [Deploy 1Password SCIM Bridge on Google Cloud Platform](https://support.1password.com/scim-deploy-gcp/)
 - [Deploy 1Password SCIM Bridge on DigitalOcean](https://support.1password.com/scim-deploy-digitalocean/)
 
-## Advanced deployment
+## Custom deployment
 
-If you have particular requirements for your environment, an advanced deployment is recommended. These example configurations can be used as a base for a customized deployment.
+For a more customizabile deployment, whether you choose a container-as-a-service option to allow the infrastructure to be managed by your service provider, or an advanced deployment, you can use these example configurations as a base for a customized deployment.
 
 ### Before you begin
 
 Before you begin deploying 1Password SCIM Bridge, review the [Preparation Guide](/PREPARATION.md). The guide will help you plan for some of the technical components of the deployment and consider some issues you may encounter along the way.
 
-After you've read the preparation guide, refer to the deployment example for your infrastructure:
+### Container-as-a-Service deployment options
+
+Container-as-a-Service (CaaS) deployments of the 1Password SCIM Bridge will simpify your deployment by utilizing the built-in tools of the CaaS service for DNS and certificate management. This gives you an easy, low cost SCIM bridge with few instrastructure management requirements.
```

Comment:
To match the suggestions above and make this a tad more concise. I also changed "few" to "minimal" since that clicks in my mind as "ooo this is going to be nice!"

```suggestion
### Containers as a service deployment options

Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
```